doc/releases.md

```diff
@@ -32,7 +32,7 @@ Release builds require manual promotion by an individual with SSH access to the
 
 A SHASUMS256.txt file is produced for every promoted build, nightly, and releases. Additionally for releases, this file is signed by the individual responsible for that release. In order to be able to verify downloaded binaries, the public should be able to check that the SHASUMS256.txt file has been signed by someone who has been authorized to create a release.
 
-The GPG keys should be fetchable from a known third-party keyserver. The SKS Keyservers at <https://sks-keyservers.net> are recommended. Use the [submission](https://sks-keyservers.net/i/#submit) form to submit a new GPG key. Keys should be fetchable via:
+The GPG keys should be fetchable from a known third-party keyserver. The SKS Keyservers at <https://sks-keyservers.net> are recommended. Use the [submission](http://pgp.mit.edu:11371/#submit) form to submit a new GPG key. Keys should be fetchable via:
```

Comment:
>The SKS Keyservers at <https://sks-keyservers.net> are recommended.

Should this part be changed?

Reply:
I think it's fine. The MIT server is in the [pool](https://sks-keyservers.net/status/).